tests/test_log_to_statsd.py

```diff
@@ -15,8 +15,10 @@ def test_log_to_statsd(self, mock_stats_client):
             }
         }
         log_to_statsd(mock_client, 'prefix', results)
-        mock_client.timing.assert_called_once_with('stats.timer.prefix.example_service.http://example.com.200', 100)
-        mock_client.incr.assert_called_once_with('stats.counter.prefix.example_service.http://example.com.200')
+        mock_client.timing.assert_called_once_with(
+            'stats.timer.prefix.example_service.http://example.com.200', 100)
+        mock_client.incr.assert_called_once_with(
+            'stats.counter.prefix.example_service.http://example.com.200')
```

Comment:
there is no error with this double slashes in statsd? did you check it?

Reply:
maybe better to remove part with http:// ?

Reply:
also statsd will plit example and com

Reply:
Not when i tested.